#standard metadata for higher-level flow data types
uint32 seq
time stamp
string frame_id
int16[7] Current
int16[7] Velocity
int64[7] Position
int8[7] Online_Check
int8[7] Ctl1_Value
int8[7] Ctl2_Value
